xml/containers-support.xml

```diff
@@ -218,7 +218,7 @@
        &suse; supports components that come from the &suse; base containers.
        Packages from &suse; repositories are also supported. Additional
        components and applications in the containers are not covered by &suse;
-       support. A &slea; subscription is required for building a derived
+       support. An &slea; subscription is required for building a derived
```

Comment:
This was right the first time, 'A &slea;'

Reply:
I thought the same, TBH. The question is whether one pronounces "SLE" as "slee" or "ess ell ee". Both are viable; cf. SQL → sequel or ess cue ell.